src/react-apps/applications/ux-editor/src/components/config/SelectionEditComponent.tsx

```diff
@@ -0,0 +1,284 @@
+import { createStyles, FormControlLabel, Grid, IconButton, Input, Radio, RadioGroup, Typography, withStyles } from '@material-ui/core';
+import * as React from 'react';
+import { connect } from 'react-redux';
+import Select from 'react-select';
+import altinnTheme from '../../../../shared/src/theme/altinnStudioTheme';
+import { getCodeListIndexByName } from '../../utils/apiConnection';
+import { noOptionsMessage, renderSelectDataModelBinding, renderSelectTextFromResources } from '../../utils/render';
+import { customInput } from './EditModalContent';
+
+const styles = createStyles({
+  inputHelper: {
+    marginTop: '1em',
+    fontSize: '1.6rem',
+    lineHeight: '3.2rem',
+  },
+  formComponentsBtn: {
+    'fontSize': '0.85em',
+    'fill': altinnTheme.altinnPalette.primary.blueDarker,
+    'paddingLeft': '0',
+    'marginTop': '0.1em',
+    'outline': 'none !important',
+    '&:hover': {
+      background: 'none',
+    },
+  },
+  specialBtn: {
+    fontSize: '0.6em !important',
+  },
+  formControlLabel: {
+    fontSize: '1.6rem',
+    paddingRight: '58px',
+  },
+  radio: {
+    color: ' #0095DD' + '!important',
+  },
+  text: {
+    fontSize: '1.6rem',
+  },
+  textWithTopPadding: {
+    fontSize: '1.6rem',
+    paddingTop: '24px',
```

Comment:
Nope, I'll update with consistent values.

Reply:
Some overlap between rem and em still, I coped the em units from your styling of the delete icon, so I'll leave that untouched. Otherwise unites are now consistent.